bc_obps/registration/api/user_operator.py

```diff
@@ -389,9 +416,7 @@ def update_operator_and_user_operator(request, payload: UserOperatorOperatorIn,
 @authorize(AppRole.get_all_authorized_app_roles(), ["admin"])
 def update_user_operator_status(request, payload: UserOperatorStatusUpdate):
     current_user: User = request.current_user  # irc user or industry user admin
-    if payload.user_guid:  # to update the status of a user_operator by user_guid
-        user_operator = get_object_or_404(UserOperator, user_id=payload.user_guid)
```

Comment:
As we no longer need the `user_guid`, I believe we can remove it from the UserOperatorStatusUpdate schema and a couple of places in front-end.